### PR TITLE
initial draft

### PR DESCRIPTION
**Pull request by ipc103**

### CODE REVIEW

File: js/app/services/UserService.js [hunk 1]
```diff
@@ -0,0 +1,13 @@
+function UserService($http) {
+	this.getUser = function () {
```
Comment (ipc103) [inline js/app/services/UserService.js:2]: I'm unclear on how to refactor this using `$resource`. My thinking would be something like this: 

``` js
function UserService($resource) {
  return $resource('/rest/user/:userId')
}
```

But this wont attach a method called  `getUser` - should I attach those properties to the UserService still?

**Comment by ipc103** [comment]
Love the idea of refactoring a service using `$http` into one using `$resource`. We need to include tests so that the users can make sure this doesn't break anything. Is it possible to test that they're using `$resource` instead of `$http` and that the methods haven't changed? 

I was also a little unclear on how to implement the `$resource` into a service - can you show an example of this in the previous README?

**Comment by toddmotto** [comment]
Unfortunately it is quite hard to test the usage of $resource! I'll sort out the README example to show how to use $resource in a service.

**Comment by ipc103** [comment]
Checked out the previous readme - clarified a lot! There is no way to test Resource, so leaving this without specs...

**Pull request merged**